lib/rust/ensogl/core/src/display/world.rs

```diff
@@ -259,8 +259,10 @@ impl WorldData {
                 } else if key == "Digit2" {
                     display_mode.set(2)
                 } else if key == "KeyP" {
-                    let log = profiler::internal::take_log();
-                    web_sys::console::log_1(&log.into());
+                    enso_debug_api::save_profile(&profiler::internal::take_log());
+                } else if key == "KeyQ" {
```

Comment:
Why the "q" button? I'm always afraid of q-related shortcuts, because `cmd + q` closes the application without asking about the confirmation (standard macOS behavior that I hate).